dbc2val/CHANGELOG.md

```diff
@@ -5,4 +5,4 @@ This file lists important changes to dbc2val
 ## Refactoring and changed configuration format (2023-02)
 
 Feeder refactored and new mapping format based on VSS introduced, see [documentation](mapping.md).
-Old mapping format no longer supported.
\ No newline at end of file
+Old mapping format no longer supported.
```

Comment:
what is here different?

Reply:
Github is not that good at showing subtle differences. If you use for example gitk you will see that a newline has been added.
 
```
 Feeder refactored and new mapping format based on VSS introduced, see [documentation](mapping.md).
-Old mapping format no longer supported.
\ No newline at end of file
+Old mapping format no longer supported.
```